doc/release_cycle.md

```diff
@@ -5,8 +5,10 @@ For now we decided to go with 6 week release cycle. At the beginning of each rel
 
 Versioning is based on [Semantic Versioning 2.0.0](https://semver.org/). \
 
-Release branch annotation: `release/vX.Y.Z` \
-Release tag annotation: `vX.Y.Z`
+Release branch annotation: `release/vX.Y` (in branch name we use only `MAJOR`.`MINOR`, we don't specify `PATCH`) \
+Release tag annotation: 
+- testnet: `vX.Y.Z-beta` 
+- mainnet: `vX.Y.Z`
```

Comment:
when and on which branch do you want to create vX.Y.Z-beta tag ? In proposal we have only temporary `release/vX.Y.Z`  branch -> I guess creating tag from any commit on temporary branch is not good as these commits might be squashed, etc... After it is merged into master, we will create `vX.Y.Z` tag, which will be the same for mainnet as well as testnet ?